docs/reference/release-notes/8.0.0-beta1.asciidoc

```diff
@@ -24,6 +24,17 @@ Also see <<breaking-changes-8.0,Breaking changes in 8.0>>.
   anomaly detection results for different jobs to be mixed.
   Wait for `8.0.0-rc1` before attempting to test upgrading a cluster
   where {ml} was used prior to version `7.7.0`.
+* System indices such as `.security` or `.kibana` are not hidden
+  indices, but they will be in the GA release. As a consequence,
```

Comment:
This wording feels a bit awkward. I wonder if @bytebilly has some thoughts on how to best express this?

Reply:
I left a possible alternative version below.